apps/native/app/src/lib/my-pages-links.ts

```diff
@@ -0,0 +1,34 @@
+import { environmentStore } from '../stores/environment-store'
+
+export const getMyPagesLinks = () => {
+  const baseUrl =
+    environmentStore.getState().environment?.baseUrl ?? 'https://island.is'
```

Comment:
Should be a useMyPagesLink hook using `useEnvironmentStore(s => s.environment?.baseUrl ?? 'https://island.is')`?